### PR TITLE
HAproxy 2.0.8, Varnish 6.0.5

### PR DESCRIPTION
**Pull request by arthurpro**

### CODE REVIEW

**Comment by tmannherz** [comment]
Is the idea that we merge in https://github.com/thegrommet/docker-images/pull/7 now and then merge this PR after deploying the upgrades to production?

**Comment by arthurpro** [comment]
Yes, that's correct

**Comment by arthurpro** [comment]
but the images are already in the docker hub
I wanted to keep this open in case there will be additional fixes

File: haproxy/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
+FROM alpine
```
Comment (tmannherz) [inline haproxy/Dockerfile]: What's the benefit of basing this on `alpine` rather than `haproxy:2.06`?

Comment (arthurpro) [reply haproxy/Dockerfile]: I have compiled varnish and haproxy packages for our infra with patches, modules and config tweaks.
using official image without knowing how it was built is not going to work if we want to match our prod and dev infra.

Comment (arthurpro) [reply haproxy/Dockerfile]: and I usually use alpine to build the slimmest possible images

**Comment by arthurpro** [comment]
I updated Varnish and HAProxy images. It's time to merge this branch

**Pull request merged**